.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1 @@
+blank_issues_enabled: true
```

This could be helpful for people who aren't sure how to answer some of the questions, or might encourage asking for help troubleshooting before opening an issue.

```suggestion
blank_issues_enabled: true
contact_links:
  - name: Sopel's IRC channel
    url: irc://irc.freenode.net/#sopel
    about: Someone is almost always available to answer questions in \#sopel on freenode.
```

At least, it'll be most helpful _if_ GitHub allows `irc://` URLs. The channel and network names are given in case the link doesn't work for the user (e.g. no registered client to handle it).

---

Github does not appear to support ircs:// or irc:// =(
Changed it to freenode webchat.